core/src/main/java/io/grpc/internal/ServerCallImpl.java

```diff
@@ -192,7 +192,7 @@ ServerStreamListener newServerStreamListener(ServerCall.Listener<ReqT> listener)
   }
 
   @Override
-  public Attributes attributes() {
+  public Attributes getAttributes() {
     return stream.attributes();
```

Comment:
There's also setter in this interface - https://github.com/grpc/grpc-java/blob/master/core/src/main/java/io/grpc/internal/ServerStream.java#L83

At least it's in `internal` package, so I guess we should rename it to `getAttributes` too?